content/client-server-api/modules/end_to_end_encryption.md

```diff
@@ -1512,20 +1512,7 @@ message.
 
 The plaintext payload is of the form:
 
-```json
-{
-  "type": "<type of the plaintext event>",
-  "content": "<content for the plaintext event>",
-  "sender": "<sender_user_id>",
-  "recipient": "<recipient_user_id>",
-  "recipient_keys": {
-    "ed25519": "<our_ed25519_key>"
-  },
-  "keys": {
-    "ed25519": "<sender_ed25519_key>"
-  }
-}
-```
+{{% definition path="api/client-server/definitions/olm_payload" %}}
```

Comment:
I replaced the example with a schema because that allows us to reuse `device_keys.yaml`.

Reply:
(and it's just generally Better. We're trying to phase out these old-style json blocks)